crates/extensions/tedge_mqtt_bridge/src/lib.rs

```diff
@@ -0,0 +1,313 @@
+use async_trait::async_trait;
+use c8y_api::utils::bridge::main_device_health_topic;
+use c8y_api::utils::bridge::C8Y_BRIDGE_DOWN_PAYLOAD;
+use c8y_api::utils::bridge::C8Y_BRIDGE_UP_PAYLOAD;
```

Comment:
These definitions can be moved to `tedge_api` to avoid this dependency on `c8y_api`.